pkg/provider/pipeline.go

```diff
@@ -12,6 +12,17 @@ import (
 func dataPipeline() *schema.Resource {
 	return &schema.Resource{
 		Read: dataPipelineRead,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
According to https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.0.3/helper/schema#ResourceImporter we should be moving towards using `StateContext` here instead, which simply takes an extra `context` argument.

Similarly there's an equivalent `ImportStatePassthroughContext` that can be used with it.

Reply:
Done